Magento_Theme/templates/header/top-links/wishlist.phtml

```diff
@@ -0,0 +1,30 @@
+<?php
+/* @var \Magento\Wishlist\Block\Link $block */
+?>
+
+<a class="top-bar__links-item"
+   data-bind="scope: 'wishlist'"
+   <?= /* @noEscape */ $block->getLinkAttributes() ?>
+>
+    <svg title="heart" class="top-bar__links-icon">
+        <use xlink:href="<?= $block->getViewFileUrl('/images/icons-sprite.svg#heart'); ?>"></use>
+    </svg>
+    <?= $block->escapeHtml($block->getLabel()) ?>
+    <!-- ko if: wishlist().counter -->
+    <span data-bind="text: wishlist().counter" class="counter qty"></span>
```

Comment:
same thing as with shopping cart counter. If we can add here aria-label wiith Info about product in wishlist, it'd be great, if not, add to [#133](https://github.com/SnowdogApps/magento2-alpaca-components/issues/133)

Reply:
added to #133